src/generic/grid.cpp

```diff
@@ -3873,6 +3940,20 @@ void wxGrid::ProcessRowLabelMouseEvent( wxMouseEvent& event, wxGridRowLabelWindo
     CalcGridWindowUnscrolledPosition(0, event.GetPosition().y,  NULL, &y, gridWindow);
     int row = YToRow( y );
 
+    if ( m_cancelledDragging )
+    {
+        if ( event.LeftIsDown() )
+            return;
+        else
+            m_cancelledDragging = false;
+    }
```

Comment:
This might be a trivial function, but I'd still extract it into some helper `bool wxGrid::CheckIfDragCancelled(const wxMouseEvent& event)` and then use it like
```suggestion
    if ( CheckIfDragCancelled() )
        return;
```

If nothing else, this will allow writing a comment explaining what/why are we doing this in this function once.